docs/design/fabric-v2+/fpc-cc-invocation.puml

```diff
@@ -35,74 +35,103 @@ participant "ERCC Peer1_Org1" as ERCC1 order 50
 
 ref over User1, ERCC1
   see fpc-registration-new.puml
+  |||
 end ref
 |||
 
-activate User1
-
-User1 -> ERCC1 ++: getECCInfo(CC_NAME)
-note right User1: The user may retrieve the ECC info for a specific <CC_NAME, ...>
-return ECCInfo = List<<ECC1_S_PK, ECC1_E_PK, ... , ATTESTATION>, ...>
-
-User1 -> User1 : verify ECCInfo
-note right #FFaa00
-  Only necessary iff User1 is not member of Org1 (otherwise authenticated response enough)
-  In that case, in general user also has to query different peers from different orgs,
-  enough to satisfy a quorum as necessary for the lifecycle endorsement policy and channel
-  configuration changes.
-  In this case, though, assuming client has a reference ECC_MR, TLCC_MR, CH_id
-  it might be sufficient to perform verification done in ""verify and store"" by peers in
-  ""register attestation"" flow.
-  (TBD)
+User1 -> CLI ++: invoke <CC_NAME,..., args>
+
+group prepare FPC proposal
+
+CLI -> ERCC1 ++: query getECCInfo(CC_NAME)
+return ECCInfo = List<CC_S_PK, CC_E_PK, ... , ATTESTATION>, ...>
+
+CLI -> CLI: verify ECCInfo
+note right
+  NOTE ECCInfo contains a list of all FPC enclave registered for CC_NAME
+
+  Verification is only necessary iff User is not member of Org1,
+  otherwise authenticated response enough. In that case, the user
```

Comment:
What is actually not necessarily obvious what `verify` means, maybe mention that this means doing the attestation verification and alike (essentially what ercc does, so we could explicitly reference the part in ercc which does it and say that the client would have to replicate these steps (with the ground truth of IAS keys and tlcc of course based on some local state)

Reply:
ok added some clarification and liked to fpc-registration.puml